port_ocean/core/defaults/initialize.py

```diff
@@ -163,18 +197,25 @@ async def _create_resources(
         )
 
 
-async def _initialize_defaults(
+async def _initialize_integration(
     config_class: Type[PortAppConfig], integration_config: IntegrationConfiguration
 ) -> None:
     port_client = ocean.port_client
     defaults = get_port_integration_defaults(config_class)
     if not defaults:
-        logger.warning("No defaults found. Skipping...")
+        logger.warning("No defaults found. Skipping initialization...")
         return None
 
+    await _initialize_required_integration_settings(
+        port_client, defaults.port_app_config, integration_config
+    )
+
+    if not integration_config.initialize_port_resources:
+        return
```

Comment:
What if i expect the integration to fail when starting up if does not exist 
Should we handle this?

Reply:
not sure i got what u meant